api/handlers/service_instance_test.go

```diff
@@ -566,6 +567,29 @@ var _ = Describe("ServiceInstance", func() {
 			Expect(rr).To(HaveHTTPStatus(http.StatusNoContent))
 		})
 
+		When("the service instance is managed", func() {
+			BeforeEach(func() {
+				serviceInstanceRepo.GetServiceInstanceReturns(repositories.ServiceInstanceRecord{
+					GUID:      "service-instance-guid",
+					SpaceGUID: "space-guid",
+					Type:      korifiv1alpha1.ManagedType,
+				}, nil)
+			})
+
+			It("deletes the service instance", func() {
```

Comment:
Maybe change the description of the `It` to `responds with deletion job location`